config/initializers/devise.rb

```diff
@@ -1,6 +1,7 @@
 # Use this hook to configure devise mailer, warden hooks and so forth.
 # Many of these configuration options can be set straight in your model.
 Devise.setup do |config|
+  config.secret_key = 'dabd54db3b77f2b880cffc1e90c85270063db8c636eb3280dd0c81bf40f633c3901a5c3d0ec04cb5965c2ab9aceac8e39ea08ad6b7dd3f431d2b4651080e586b' # rubocop:disable Metrics/LineLength
```

Comment:
This value is overwritten at the end of this config file in the production ENV.
Perhaps it would be clearer if the secret key settings were together? 🐘